lib/modules/datasource/packagist/schema.ts

```diff
@@ -45,18 +40,13 @@ export const MinifiedArray = z.array(z.record(z.unknown())).transform((xs) => {
 });
 export type MinifiedArray = z.infer<typeof MinifiedArray>;
 
-export const ComposerRelease = z
-  .object({
-    version: z.string(),
-  })
-  .merge(
-    looseObject({
-      homepage: z.string(),
-      source: z.object({ url: z.string() }),
-      time: z.string(),
-      require: z.object({ php: z.string() }),
-    })
-  );
+export const ComposerRelease = z.object({
+  version: z.string(),
+  homepage: looseValue(z.string()),
```

Comment:
what happens if the server sends response where those fields are missing instead of null or string?

Reply:
It adds them as `null`

Reply:
ok, just wanted to make sure it's not breaking again 🫣